### PR TITLE
More Secured Password Input

### PR DESCRIPTION
**Pull request by asaf400**
Short Description: 
More Secured Password Input (interactive input instead of command-line [which might have history])
Still Support command-line argument, but if no password is given in arguments, the config initializing stage asks for it.

### CODE REVIEW

File: pokecli.py [hunk 1]
```diff
@@ -75,7 +77,11 @@ def init_config():
     if config.auth_service not in ['ptc', 'google']:
       log.error("Invalid Auth service specified! ('ptc' or 'google')")
       return None
-
+      
+    if not config.password :
+        #log.warn("Password not passed as argument, Asking User for input")
+        config.password = getpass.getpass("Please enter You're Password: ")
```
Comment (douglascamata) [inline pokecli.py]: fix the typo, please, and I'll happily merge it.

**Comment by douglascamata** [comment]
And rebase your branch from `dev` too. Thanks!

**Comment by asaf400** [comment]
Will do (I don't have much time atm, but will try..)

**Comment by asaf400** [comment]
so for now cancel this request right?

**Comment by douglascamata** [comment]
nope, just `go pull` from our dev branch and fix the conflict.

**Comment by asaf400** [comment]
Someone (rarshonsky) has bitten me to it :)

https://github.com/PokemonGoF/PokemonGo-Bot/commit/cf0bab5e85ea369962f22ef26a6bf7b8e9f59d21

**Pull request closed without merging**